docs/build.gradle.kts

```diff
@@ -14,11 +16,35 @@ dependencies {
     implementation(libs.opentelemetry.exporter.logging)
 }
 
+val properties = Properties().apply {
+    load(file("knit.properties").inputStream())
+}
+
+val knitDirectory = properties["knit.dir"] ?: "src/main/kotlin"
```

Comment:
minor: with `!!` you can make an implicit check that knit.properties is found and read successfully